torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -485,6 +485,14 @@ def aot_dispatch_autograd(
                 inner_meta.bw_donated_idxs = fw_metadata.bw_donated_idxs
 
         if aot_config.enable_log:
+            trace_structured(
```

Comment:
I think this is only going to run in the training path, but not the inference path? You might need the same treatment in `aot_dispatch_base`